client/app/bundles/shared/components/Blockquote.jsx

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import './Blockquote.css';
+
+export default class Blockquote extends React.Component {
+  render() {
+    return (
+      <div className="Blockquote">      
+        
+        <blockquote>"It's not just all in your head, it's all around you. We 
+          can heal together."</blockquote>
+          <blockquote>{"\u2764\uFE0F"}</blockquote>
+        
+      </div>
```

Comment:
A few things here!

1. Let's make sure this is just one blockquote element and not multiple.
2. Let's pass in the text as props.
3. Let's remove quotes from the JSX here and instead have the CSS generate them using pseudo elements!